app/forms/user/create.rb

```diff
@@ -0,0 +1,39 @@
+class User < ApplicationRecord
+  class Create
```

Comment:
Since my last comment got eaten by a refactor...

Notice that we have separate form objects, rather than directly using ActiveRecord model objects in our views. This, in conjunction with repositories, allows for easier testing (since you don't have to load the database) and more customization for different model actions (create vs update vs whatever else)